ecommerce/resources/views/auth/login.blade.php

```diff
@@ -0,0 +1,73 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div class="card">
+                <div class="card-header">{{ __('Login') }}</div>
```

Comment:
```suggestion
                <div class="card-header">{{ trans('Login') }}</div>
```
dùng trans hoặc lang đi em